src/backend/distributed/commands/schema.c

```diff
@@ -134,3 +138,67 @@ PlanAlterObjectSchemaStmt(AlterObjectSchemaStmt *alterObjectSchemaStmt,
 
 	return NIL;
 }
+
+
+/*
+ * EnsureSchemaExistsOnAllNodes connects to all nodes with citus extension user
+ * and creates the schema of the given relationId. The function errors out if the
+ * command cannot be executed in any of the worker nodes.
+ */
+void
+EnsureSchemaExistsOnAllNodes(Oid relationId)
+{
+	List *workerNodeList = ActivePrimaryNodeList();
+	ListCell *workerNodeCell = NULL;
+	StringInfo applySchemaCreationDDL = makeStringInfo();
+
+	Oid schemaId = get_rel_namespace(relationId);
+	const char *createSchemaDDL = CreateSchemaDDLCommand(schemaId);
+	uint64 connectionFlag = FORCE_NEW_CONNECTION;
```

Comment:
Do we need to force opening a new connection or could we just use whatever connection is open?

Reply:
After discussing this offline, we decided to keep forcing for new connections. Unless we have this, a user can not distribute more than one table in a single transaction. (The second `create_distributed_table()` call fails as the previous one ran a DDL on the connection)